src/main.cpp

```diff
@@ -4134,20 +4173,20 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         return true;
     }
 
-
-    if (!(nLocalServices & NODE_BLOOM) &&
-              (strCommand == NetMsgType::FILTERLOAD ||
-               strCommand == NetMsgType::FILTERADD ||
-               strCommand == NetMsgType::FILTERCLEAR))
-    {
-        if (pfrom->nVersion >= NO_BLOOM_VERSION) {
-            Misbehaving(pfrom->GetId(), 100);
-            return false;
-        } else if (GetBoolArg("-enforcenodebloom", false)) {
-            pfrom->fDisconnect = true;
-            return false;
-        }
-    }
+// BUIP010 Extrem Thinblocks:  We need bloom filtering. We do not turn bloom filtering off
```

Comment:
You are commenting on very old code that has been changed quite a lot since.  This specific part has been re-added in a later commit by the guy that originally removed it ;)

If you want to review BitcoinClassic I suggest you look at the develop branch, not at a couple months old commits.

Notice that this port was marked as work-in-progress. It was worked on and the actual tree change is substantially smaller as a result.